install-monorepo.sh

```diff
@@ -22,33 +22,37 @@ fi
 OS="$(uname)"
 if [[ "$OS" == "Linux" ]]; then
 	ON_LINUX=1
+	if ! command -v apt &>/dev/null; then
+		abort "Installer script requires `apt` to ensure essentials are installed."
+	fi
+	apt update
 	sudo apt install build-essential
 elif [[ "$OS" != "Darwin" ]]; then
 	abort "Installer script is only supported on macOS and Linux."
 fi
 
+# Check for curl and git
+if ! command -v git &>/dev/null; then
+	abort "Installer script requires `git` to be installed."
+fi
+
+if ! command -v git &>/dev/null; then
+	abort "Installer script requires `cURL` to be installed"
```

Comment:
Same as above about backticks.

Also, while the name of the program is "cURL", the name of the binary is "curl". Since backticks suggest the binary, I'd expect the lowercase version.